doc/lua_api.txt

```diff
@@ -6204,45 +6204,52 @@ Sorted alphabetically.
 `AreaStore`
 -----------
 
-A fast access data structure to store areas, and find areas near a given
-position or area.
-Every area has a `data` string attribute to store additional information.
-You can create an empty `AreaStore` by calling `AreaStore()`, or
-`AreaStore(type_name)`. The mod decides where to save and load AreaStore.
-If you chose the parameter-less constructor, a fast implementation will be
-automatically chosen for you.
+AreaStore is a data structure to calculate intersections of 3D cuboid volumes
+and points. The `data` field (string) may be used to store and retrieve any
+mod-relevant information to the specified area.
+
+Despite its name, mods must take care of saving and loading AreaStore data using
+the provided load and write functions.
+
 
 ### Methods
 
-* `get_area(id, include_borders, include_data)`
+* `AreaStore(type_name)`
+    * Returns a new AreaStore instance
+    * SpatialIndex will be used by default to optimize speed.
+    * `type_name`: optional, forces the internally used API.
+        * Possible values: `"LibSpatial"`.
+        * Other values will make use of the custom Minetest functions.
+* `get_area(id, include_corners, include_data)`
     * Returns the area information about the specified ID.
     * Returned values are either of these:
 
             nil  -- Area not found
-            true -- Without `include_borders` and `include_data`
+            true -- Without `include_corners` and `include_data`
```

Comment:
So what. Edges, corners or borders? All three are supposed to refer to the `minp`, `maxp` coordinates of a given cuboid, and the terms should be kept consistent to avoid confusion.

Reply:
Keep it at borders when it's about intersections - keep `include_borders` the way it was. `include_borders` does not refer only to these coordinates.

Reply:
> include_borders does not refer only to these coordinates.

It literally does mean exactly that:
https://github.com/minetest/minetest/blob/a8707158a5be8c604fdb028a9c5f68ce5804016e/src/script/lua_api/l_areastore.cpp#L49-L54

Reply:
Indeed, nevermind then. I was confusing this with `accept_overlap` it seems.